packages/graphql-testing/src/test/createGraphQLFactory.test.tsx

```diff
@@ -66,9 +69,9 @@ describe('graphql-testing', () => {
 
     await graphQL.resolveAll();
     somePage.update();
-
     expect(
       somePage.containsMatchingElement(
+        // eslint-disable-next-line shopify/jsx-no-hardcoded-content
```

Comment:
couldn't get this one to pass /shrug

Reply:
We probably never actually care about this rule in tests anyway /shrug

Reply:
We should disable this rule for the whole repo, or at least for tests, it doesn't make sense in our context.